### PR TITLE
Milestone: Completed desktop version of the portfolio. 

### PR DESCRIPTION
**Pull request by Arslan49sp**
In this branch, I managed to make this project fully responsive with respect to desktop devices.
I designed all the sections in this portfolio with respect to the desktop. 

- [ ] This branch doesn't have any conflict with the main branch.
- [ ] Does not contain any linter errors
- [ ] Used best programming practices
- [ ] Fixed all the changes that were notified by the first reviewer.
- [ ] Fixed the scrolling problem as per suggested by the second reviewer.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -423,3 +388,280 @@ hr {
...
+    height: 128px;
+    bottom: -0%;
+    left: -72%;
+    z-index: -1;
+  }
+
+  .aboutme_box_container {
+    flex-direction: row;
+    justify-content: center;
+  }
```
Comment (DuaneDave) [inline styles.css:595]: Consider working on the layout of this section to make it adaptable to the chosen media breakpoint. Currently all your cards break into one column and in some larger breakpoints two column which is not in accordance with the design guidelines. I suggest fixing this in order to make you design similar to the original template which is always three columns beginning from the desktop and one column on mobile.

one column 
hmmm

Two column
ds

HINT: I believe the width on the cards is still the cause of this problem. You should always use 100% for thing you'd want to  be able to resize as the screen changes

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @Arslan49sp 👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good job with the design
✔️ The page is well organized

## Required Changes ♻️
_**NOTE**_: For all changes, refer to the figma template to have a clear understanding of the page layout 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -290,27 +303,21 @@ <h2>Skills</h2>
       </ul>
     </section>
 
-    <section class="contact_container">
-      <div class="section_width">
-        <h1 class="p7s">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+    <section id="contact" class="contact_container">
+      <div class="form">
+        <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
         <form class="form_container" action="https://formspree.io/f/mzbqodyb" method="POST">
```
[the remark this answers is not in the document]
Comment (emmiiorji) [reply index.html:309]: > Good job building the mobile version of your website, however, there is an issue with the decoration on the page. They aren't positioned correctly and as a result of this, it is causing some horizontal scroll to the left across the page. Maybe you should try adding an `overflow: hidden` property on the body to resolve it or try playing with the position property in order to make the UI much straightforward and nicer.
> 
> over

As requested by the previous reviewer and in order to imporve the user experience, kindly adjust the design to remove the horizontal scroll.

image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Arslan49sp, 

# Status: Changes Required ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<div align='center'>

</div>

 ## Good Highlights✔️

- The skills section no longer wraps as requested

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi @Arslan49sp , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Good design👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**